modules/tinymce/src/plugins/table/main/ts/actions/EnforceUnit.ts

```diff
@@ -7,10 +7,10 @@
 
 import { Arr } from '@ephox/katamari';
 import { Css, Element, SelectorFilter, Traverse } from '@ephox/sugar';
-import { getPixelWidth } from '../alien/Util';
 import { HTMLTableElement } from '@ephox/dom-globals';
+import { CssUtils } from '@ephox/snooker';
 
-const calculatePercentageWidth = (element: Element, parent: Element): string => getPixelWidth(element.dom()) / getPixelWidth(parent.dom()) * 100 + '%';
+const calculatePercentageWidth = (element: Element, parent: Element): string => CssUtils.getPixelWidth(element.dom()) / CssUtils.getPixelWidth(parent.dom()) * 100 + '%';
```

Comment:
This should probably just use `Width.get()` from Sugar instead.

Reply:
Hm. Does it matter that that'd switch it from `elm.getBoundingClientRect().width` to `offsetWidth` or whatever Sugar is doing?

Reply:
Right... I looked at `Height` originally sorry which does use `getBoundingClientRect()` by default.

Reply:
> Most of the time these are the same as width and height of Element.getBoundingClientRect(), when there aren't any transforms applied to the element. In case of transforms, the offsetWidth and offsetHeight returns the element's layout width and height, while getBoundingClientRect() returns the rendering width and height

From https://developer.mozilla.org/en-US/docs/Web/API/CSS_Object_Model/Determining_the_dimensions_of_elements

So in all honesty, I'd say `Width` should be doing the same as `Height`. Not sure if you want to change that now though. Either way, I'd say it'd be safe to use `Width` here, as it's highly unlikely to be using a transform on a table element.

Reply:
I'll switch to `Width` here and make a Jira for investigating the other part
- [] make Jira to look at changing `Width` in Sugar